go/runtime/task_term.go

```diff
@@ -93,6 +101,7 @@ func (t *taskTerm) initTerm(shard consumer.Shard, host *FlowConsumer) error {
 		"initialized catalog task term",
 		"labels", t.labels,
 		"lastLabels", lastLabels,
+		"reactorInstance", t.consumerConfig,
```

Comment:
Rather than threading through the service config, you can instead use the [Shard.Assignment](https://github.com/gazette/core/blob/master/consumer/interfaces.go#L47-L50) accessor to get the [allocator.Assignment](https://github.com/gazette/core/blob/master/allocator/allocator_key_space.go#L72), which has this info and a bit more:

```
// The cast isn't even technically necessary if you're just logging `Decoded`.
"assignment": shard.Assignment().Decoded.(allocator.Assignment)
```

Reply:
Looks like that works even better, since hostname in consumer config is just a pod IP, and the `MemberSuffix` on the assignment is the k8s pod name. Easiest thing was to just re-implement as a new commit, so I'm just closing this in favor of #1039